test/unit/logic/community-voice.spec.ts

```diff
@@ -249,14 +301,58 @@ describe('Community Voice Logic', () => {
             )
             expect(mockPubsub.publishInChannel).toHaveBeenCalledWith(COMMUNITY_VOICE_CHAT_UPDATES_CHANNEL, {
               communityId,
-              status: 'started'
+              status: 0, // ProtocolCommunityVoiceChatStatus.COMMUNITY_VOICE_CHAT_STARTED
+              positions: ['1,1', '1,2', '2,1', '2,2'],
+              communityName: 'Test Community',
+              communityImage: 'test-community.jpg'
             })
             expect(mockAnalytics.fireEvent).toHaveBeenCalledWith(AnalyticsEvent.START_COMMUNITY_CALL, {
               call_id: communityId,
               user_id: creatorAddress
             })
           })
         })
+
+        describe('when getCommunityPlaces fails', () => {
+          beforeEach(() => {
+            mockCatalystClient.getProfile.mockResolvedValue(createMockProfile(creatorAddress))
+            mockCommunitiesDb.getCommunityPlaces!.mockRejectedValue(new Error('Places fetch failed'))
+          })
+
+          it('should successfully start a community voice chat without positions when fetch fails', async () => {
+            const result = await communityVoice.startCommunityVoiceChat(communityId, creatorAddress)
+
+            expect(result).toEqual({ connectionUrl: 'test-connection-url' })
+            expect(mockPubsub.publishInChannel).toHaveBeenCalledWith(COMMUNITY_VOICE_CHAT_UPDATES_CHANNEL, {
+              communityId,
+              status: 0, // ProtocolCommunityVoiceChatStatus.COMMUNITY_VOICE_CHAT_STARTED
+              positions: [],
+              communityName: 'Test Community', // Still gets community info even when places fail
+              communityImage: 'test-community.jpg'
+            })
+          })
+        })
+
+        describe('when placesApi fails', () => {
+          beforeEach(() => {
+            mockCatalystClient.getProfile.mockResolvedValue(createMockProfile(creatorAddress))
+            mockCommunitiesDb.getCommunityPlaces!.mockResolvedValue([{ id: 'place-1' }])
+            mockPlacesApi.getPlaces.mockRejectedValue(new Error('PlacesApi failed'))
+          })
+
+          it('should successfully start a community voice chat without positions when placesApi fails', async () => {
```

Comment:
Would you mind removing the redundant `when placesApi fails` here?

```suggestion
          it('should successfully start a community voice chat without positions', async () => {
```